components/Translation/basic_usage.rst

```diff
@@ -21,12 +21,12 @@ First let's create a class which will keep our translatable properties:
        /**
         * @var string
         */
-       private $title;
+       private function $title;
```

Comment:
Overall good catch with missing `function`'s, but you took a little too far, this isn't a function, but a variable.
Other than that :+1: